src/js/components/covid19/Covid19Page.jsx

```diff
@@ -64,6 +66,15 @@ const Covid19Page = ({ areDefCodesLoading }) => {
         }
     }, [history, isRecipientMapLoaded, query]);
 
+    useEffect(() => {
```

Comment:
`query.publicLaw === 'american-rescue-plan'` should get you the same info without having to import `useLocation` or define a state variable. ([useQueryParams](https://github.com/fedspendingtransparency/usaspending-website/blob/qat/src/js/helpers/queryParams.js) is a custom hook we created for use cases like this where we want to access URL params as object properties)

Reply:
Thank you for pointing this out! Working on refactoring to use this hook